tests/ci/android/README.md

```diff
@@ -25,3 +27,17 @@ Emulators are not easily configurable out of the box on Linux, and it would be b
 3. Go to the Tools menu -> Android -> AVD Manager. If no emulator has been created you should click the "Create Virtual Device" button.
 4. Select "Pixel 2" (or any of the newer preferred options), then select "x86" images and download R (API Level:30, ABI:x86_64, Android 11.0 (Google APIs)). Click through the rest of the virtual device set up and then "Finish". The newly created device should be launchable from the AVD Manager window.
 5. Run `./gradlew cC` on `AWSLCAndroidTestRunner`, and the emulator will be automatically detected. Outputted debug logs from the emulator device can be seen from the log interface in any blank project created in Android Studio. You can also use `awslc-test` tag to filter through specific logs outputted from running `AWSLCAndroidTestRunner`.
+
+## Setup Device Farm CI Resources
+Although the Android CI's codebuild resources are integrated with our current CI infrastructure, it also relies on additional Device Farm resources to run the real device tests after cross-compiling within Codebuild. Using the cdk action type `update-android-resources` will create the AWS Device Farm resources the CI needs. For steps on how to use the cdk script, refer to `tests/ci/cdk/README.md`.
```

Comment:
Can the AWS Device Farm resources be created via CDK/Cloudformation? After the creation, can these resources be reused? Or the device farm resources are expected to be re-created in each CodeBuild job?